test/react-native-cli/features/built-app.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Tests against a React Native app that was initialized using the Bugsnag React Native CLI
+
+Scenario: Build app sends JavaScript and Native handled errors
+  When I notify a handled JavaScript error
```

Comment:
I can't see any configuration that prevents sessions from sending, so I would have expected a session to be the first payload that's delivered?

It seems like this is a potential avenue for flakes, seeing as the order of delivery might be non-deterministic. I think that checking for the delivery of a session is another useful data point in determining whether the app was configured correctly, so I'd lean heavily in favour of that over preventing sessions from being sent (which is the approach taken in many of our end-to-end tests).

Reply:
I completely agree and there's some going on here that I don't yet understand.  I'm not seeing a session request at all currently, even if I just sit and wait for two minutes.

Reply:
This was due to MazeRunner clearing the set of received in the Cucumber `Before` hook - which happens after the RN CLI test fixture is first opened and the session request sent.  MazeRunner has now been updated to reset its list of received requests in the `After` hook, avoiding this situation whilst still achieving its core objective.